services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -111,11 +115,13 @@ const Asset = ({ libraryData }) => {
                   <dt className={dashboardStyles.label}>Library</dt>
                   <dd className={dashboardStyles.labelLarge}>{libraryData.content.name}</dd>
                 </dl>
-                <Link href={libraryPath}>
-                  <a className={clsx(dashboardStyles.metaLink, dashboardStyles.metaLinkLarge)}>
-                    {`v${libraryData.content.version}`}
-                  </a>
-                </Link>
+                <CarbonLink
```

Comment:
Tabs look great! I think we need to wrap the CarbonLink in the Next js Link component, is that correct @mattrosno 

I actually just opened a small PR here to fix the styles https://github.com/carbon-design-system/carbon-platform/pull/407

Reply:
and so in this case - yep they would need to be wrapped. I'll remove my update from this PR so it doesn't conflict with yours, which is the correct solution.  :)